frontend_v2/src/app/components/utility/modal/modal.component.ts

```diff
@@ -36,9 +36,6 @@ export class ModalComponent implements OnInit {
 
   /**
    * Invalid fields
-   */
-  invalidFields = [];
```

Comment:
@Alabhya268 why are we removing this field

Reply:
I may have done it by mistake. I'll fix it instantly.